src/common/options/global.yaml.in

```
@@ -3092,6 +3092,13 @@ options:
   default: 5
   fmt_desc: How many operations logs to display at once.
   with_legacy: true
+- name: osd_op_thread_suicide_timeout
+  type: int
+  level: advanced
+  default: 150
+  fmt_desc: a configuration value used to make the OSD suicide when operations
```

Propose to capitalize the first letter in line with other (well, some of the) entries in the file.

```suggestion
  fmt_desc: A configuration value used to make the OSD suicide when operations
```

It would be interesting to know what this "suicide" actually means/when does it happen which was the motivation for my opening of the issue. But I understand this is not the place for such explanations and perhaps I should not worry about value of this setting unless actually encountering a problem with it.

---

@bluikko: This PR is the beginning of the process of explaining osd_op_thread_suicide_timeout, not the end. I share your desire for more information about this timeout: what conditions, for example, should make an experienced Ceph administrator change its value? I won't be closing https://tracker.ceph.com/issues/58590 until there's good, reliable guidance regarding osd_op_thread_suicide_timeout. 

I raised this PR when I did because I wanted to test the behavior of the ceph/src/ directory. That directory is outside my everyday bailiwick, and I am not familiar with its limitations in the way that I am familiar with the limitations of the .rst files in ceph/doc. This paragraph is probably mostly a note to my future self, but I should say that I learned today that one of the limitations of the yaml.in file is that it accepts URLs of only forty characters or less. I learned this when I tried to direct readers to the URL below, which Neha Ojha informed me of: 

https://old.ceph.com/planet/dealing-with-some-osd-timeouts/